### PR TITLE
Add WidgetSDK documentation workflow

### PR DESCRIPTION
**Pull request by rasmustautsglia**
**Jira issue:**
https://glia.atlassian.net/browse/MOB-3137

**What was solved?**
This PR adds the necessary script for Bitrise to execute so that documentation page can be automatically generated and uploaded during release of the Widget SDK.

MOB-3137

**Release notes:**

 - [ ] Feature
 - [X] Ignore
 - [ ] Release notes (Is it clear from the description here?)
 - [ ] Migration guide (If changes are needed for integrator already using the SDK - what needs to be communicated? Add underneath please)

**Additional info:**

 - [ ] Tests fixed, added? Unit, acceptance, snapshots?
 - [ ] Logging necessary for future troubleshooting of customer issues added?

**Screenshots:**

Widgets doc page:
http://ios-sdk-docs.salemove.com.s3-website-us-east-1.amazonaws.com/widgets/index.html

### CODE REVIEW

**Review by yurii-glia — APPROVED** [review]

File: .jazzy.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# You can find build configs and their default values here:
+# https://github.com/realm/jazzy/blob/7e8a61a4f1d18ac155a1325572a24ec86e57549f/lib/jazzy/config.rb#L118
+clean: true
+author: Glia Technologies, Inc
+author_url: https://glia.com
+github_url: https://github.com/salemove/ios-sdk-widgets
+root_url: https://docs.glia.com/glia-mobile/docs/ios-widgets-sdk
```
Comment (yurii-glia) [inline .jazzy.yaml:7]: should this url point to docs or to our guides?

**Review by EgorovEI — APPROVED** [review]

File: .jazzy.yaml (hunk 1 above)
Comment (gugalo) [reply .jazzy.yaml:7]: In jazzy it is described as:
> Absolute URL root where these docs will be stored

Will change the URL to point to the API docs themselves.

Comment (gugalo) [reply .jazzy.yaml]: Fixed in 207120b

**Comment by gugalo** [comment]
I have no idea why Bitrise fails with error
> Failed to process Step inputs:
  simulator UDID lookup failed:
    runtime (com.apple.CoreSimulator.SimRuntime.iOS-17-5) is incompatible with device type (com.apple.CoreSimulator.SimDeviceType.iPhone-8-Plus)

**Review by igorkravchenko — APPROVED** [review]

**Comment by gugalo** [comment]
!squash

**Pull request merged**